onnxruntime/core/providers/openvino/ov_versions/capabilities.h

```diff
@@ -2,26 +2,25 @@
 // Licensed under the MIT License
 
 #pragma once
+#include<vector>
+#include "data_ops.h"
 
 namespace onnxruntime {
 namespace openvino_ep {
 
+class GetCapability {
+    private:
+        const GraphViewer& graph_viewer_;
+        std::string device_type_;
+        DataOps* data_ops_;
+    public:
+        GetCapability (const GraphViewer& graph_viewer_param, std::string device_type_param, const std::string version_param);   
+        virtual std::vector<std::unique_ptr<ComputeCapability>> Execute();
+};
+
 #if defined OPENVINO_2020_3
 std::vector<std::unique_ptr<ComputeCapability>>
 GetCapability_2020_3(const GraphViewer& graph_viewer, const std::string device_type);
```

Comment:
why 2020.3 was not converted to this new method? are we going to deprecate it soon?